client/src/commands/UserInputUtil.ts

```diff
@@ -427,4 +427,16 @@ export class UserInputUtil {
         return vscode.window.showQuickPick(quickPickItems, quickPickOptions);
     }
 
+    public static async showTestFileOverwriteQuickPick(prompt: string): Promise<string | undefined> {
+        const options: Array<string> = [this.YES, this.NO, 'generate copy'];
```

Comment:
should this be `generate new` rather than `generate copy` as you aren't copying the file you are creating a new generation